src/widget/wwidgetgroup.cpp

```diff
@@ -215,5 +215,6 @@ int WWidgetGroup::getHighlight() const {
 void WWidgetGroup::setHighlight(int highlight) {
     m_highlight = highlight;
     update();
+    style()->polish(this);
```

Comment:
[QWidget documentation](http://doc.qt.io/qt-4.8/qwidget.html#update) suggests calling update() is better than directly calling repaint().